### DISCUSSION TITLE
Allow writing in markdown, with a preview next to it

### DISCUSSION THREAD

**Discussion by sebouh00** [GitbookIO/community#156] [Feature Requests]
### What's your feature/idea?

Split the page editor in the middle and allow directly writing in markdown. Working with the editor is slow, specially when the page content is large. It's also difficult to copy paste items, such as tables. In markdown, updates can be done much faster, with the preview next to it to view the change.

### What's your desired outcome?

To work much faster and with much less frustration due to slow speed and inability to copy paste repetitive content (such as formatted class method definitions).

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by taszap** [comment]
Hi @sebouh00 

Thank you for your post. 
I may have missed some context but why not enable [Git Sync](https://docs.gitbook.com/product-tour/git-sync)? 
This should allow you to write in Markdown.

**Comment by ahcsco** [comment]
It's a good suggestion to have source code editor available without requiring git sync. I have noticed and raised to Gitbook of performance issues in Gitbook native editor when working with relatively bigger content (as in normal size of text) and moving it around. It just starts behaving really strange. At least now i know other people are also facing it.

**Comment by sebouh00** [comment]
Indeed. Editing the markdown and seeing a live preview right next to it would make it much easier to work with. I'd pretty much mostly use the markdown editor then for its speed and simplicity.

**Comment by taszap** [comment]
Thanks for sharing your feedback. Although GitBook is in many ways similar to Markdown, we do not use Markdown. Whenever your content is synced via Git Sync the content is translated with our parser. 

Your comments are still very valid though - we are aware that the editor requires improvements in terms of responsiveness and reliability and are working to address this. We will keep you updated with the developments here and in our changelog. I hope that offers some clarity before we are able to communicate on large scale improvements
